divisible.rb

```diff
@@ -0,0 +1,11 @@
+def return_array
+    @new_array = []
+    for number_divisible_by_2_or_3_or_5 in 1..100 do
+        if number_divisible_by_2_or_3_or_5 % 2 == 0 || number_divisible_by_2_or_3_or_5 % 3 == 0 || number_divisible_by_2_or_3_or_5 % 5 == 0
+            @new_array.push(number_divisible_by_2_or_3_or_5)
+        end
+    end
+    puts "#{@new_array}"
+end
+
+return_array
```

Comment:
Method names, classes and variables should all be named accordingly to what they mean.
In this case I would suggest changing the naming to something descriptive yet simple/short e.g:
```suggestion
def divisible_numbers
    @divisible_array = []
    for number in 1..100 do
        if number % 2 == 0 || number % 3 == 0 || number % 5 == 0
            @divisible_array.push(number)
        end
    end
    puts "#{@divisible_array}"
end

divisible_numbers
```